frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -203,6 +203,35 @@
             $interval.cancel(vm.logs_poller);
         };
 
+         // Get the status of workers continuously
+         vm.startFetchingWorkerStatus = function() {
+            vm.status_poller = $interval(function(){
+                parameters.url = 'challenges/' + vm.challengeId + '/get_worker_status/';
+                parameters.method = 'GET';
+                parameters.data = {};
+                parameters.callback = {
+                    onSuccess: function(response) {
+                        var details = response.data;
+                        vm.workerStatus = details.tasks_status;
+                    },
+                    onError: function(response) {
+                        var error = response.data.error;
+                        if (error == undefined){
+                            $rootScope.notify("error", "There was an error.");
```

Comment:
@burnerlee please make the error message more clear. For ex: `An error occurred when fetching worker status` or `Worker fetch status failed`